thirdparty/faiss/faiss/IndexFlatElkan.h

```diff
@@ -29,9 +29,17 @@ namespace faiss {
 // Elkan algo was introduced into Knowhere in #2178, #2180 and #2258. 
 struct IndexFlatElkan : IndexFlat {
     bool use_elkan = true;
+    float* data = nullptr;
```

Comment:
`std::vector<float> tmp_buffer_for_elkan;`

Reply:
Changed it.